File: typescript/pangram/pangram.ts
class Pangram {
  private _sentence: string

  constructor(sentence: string) {
    this._sentence = sentence
  }

  isPangram(): boolean {
    const sentence = this._sentence.toLowerCase()

    return "abcdefghijklmnopqrstuvwxyz"
      .split("")
      .filter((c) => sentence.indexOf(c) === -1)
      .length === 0
  }
}

export default Pangram

File: csharp/pangram/Pangram.cs
using System;
using System.Linq;

public static class Pangram
{
    public static bool IsPangram(string input)
        => "abcdefghijklmnopqrstuvwxyz".Count(c => input.ToLowerInvariant().IndexOf(c) == -1) == 0;
}
